libs/clients/housing-benefit-calculator/src/lib/housing-benefit-calculator.provider.ts

```diff
@@ -18,7 +18,6 @@ export const ApiConfig = {
       fetchApi: createEnhancedFetch({
         name: 'clients-housing-benefit-calculator',
         logErrorResponseBody: true,
-        treat400ResponsesAsErrors: true,
```

Comment:
I wonder if this'll break the try catch logic we have regarding receiving a 401 response

![image](https://github.com/island-is/island.is/assets/43557895/513456f7-ebeb-40c7-8046-bdbaec66b47d)

Reply:
After glancing the code a bit and taking a look at how the public vehicle search works (that has never had the treat400ResponsesAsErrors set to true) I denote that this won't affect how we are handling different status codes via try catch blocks. I've also tagged Sævar who asked for our review to confirm it in a Slack thread.

![image](https://github.com/island-is/island.is/assets/43557895/b5a9c561-7492-4c94-8304-d56400c28d1d)